assets/node-feature-discovery/05-nfd-daemonset.yml

```diff
@@ -22,7 +22,7 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: spec.nodeName
-          image: quay.io/zvonkok/node-feature-discovery:v4.0
+          image: quay.io/openshift-psap/node-feature-discovery:v0.5.0
```

Comment:
Why is this v0.5.0?

Reply:
since the PR is against master, I am pointing to match the upstream NFD operand https://github.com/kubernetes-sigs/node-feature-discovery/releases/tag/v0.5.0

Reply:
Those are openshift builds not upstream builds so please keep the v4.0 numbering

Reply:
done